exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/agg/TestHashAggrSpill.java

```diff
@@ -93,7 +93,7 @@ private void testSpill(long maxMem, long numPartitions, long minBatches, int max
   @Test
   public void testSimpleHashAggrSpill() throws Exception {
     testSpill(68_000_000, 16, 2, 2, false,
-      true, null, 1_200_000, 1, 1, 1, 3);
+      true, null, 1_200_000, 1, 2, 1, 3);
```

Comment:
This is supposed to be a "simple spill" (see comment above); not a "recursive" one (with cycle > 1).
Maybe just reducing the number of rows would do the trick ....

Reply:
Fixed. I increased the memory slightly.